types/trusted-types/index.d.ts

```diff
@@ -29,34 +23,32 @@ declare global {
         private brand: true; // To prevent structural typing.
     }
 
-    class TrustedURL {
-        private constructor(); // To prevent instantiting with 'new'.
-        private brand: true; // To prevent structural typing.
-    }
-
-    interface TrustedTypePolicy {
-        readonly name: string;
-        createHTML(input: string): TrustedHTML;
-        createScript(input: string): TrustedScript;
-        createScriptURL(input: string): TrustedScriptURL;
-        createURL(input: string): TrustedURL;
-    }
-
     interface TrustedTypePolicyFactory {
         createPolicy<Keys extends keyof TrustedTypePolicyOptions>(
-            name: string,
-            policyOptions: Pick<TrustedTypePolicyOptions, Keys>,
-            expose?: boolean,
+            policyName: string,
+            policyOptions?: Pick<TrustedTypePolicyOptions, Keys>,
         ): Pick<TrustedTypePolicy, 'name' | Keys>;
-        getPolicyNames(): string[];
         isHTML(value: any): value is TrustedHTML;
```

Comment:
I think it would be a bit more ts idiomatic if all the `is*` functions took `unknown` instead of `any`